src/server/getters/get-complete-sets.ts

```diff
@@ -0,0 +1,32 @@
+import * as Knex from "knex";
+import { Address, CompleteSetsRow } from "../../types";
+import { formatBigNumberAsFixed } from "../../utils/format-big-number-as-fixed";
+
+export interface CompleteSets {
+  [orderId: string]: CompleteSetsRow<string>;
+}
+
+export function getCompleteSets(db: Knex, account: Address, callback: (err: Error|null, result?: CompleteSets) => void): void {
+  const query = db.select("*").from("completeSets")
+    .where("account", account);
+  query.leftJoin("blocks", "completeSets.blockNumber", "blocks.blockNumber");
+  query.asCallback((err: Error|null, completeSets?: Array<CompleteSetsRow<BigNumber>>): void => {
+    if (err) return callback(err);
+    if (!completeSets) return callback(err, {});
+    callback(null, completeSets.reduce((acc: CompleteSets, cur: CompleteSetsRow<BigNumber>) => {
+      acc[cur.transactionHash] = formatBigNumberAsFixed<CompleteSetsRow<BigNumber>, CompleteSetsRow<string>>({
```

Comment:
Is transactionHash what you want it keyed by? Is that how you are looking it up?